src/netsnmpy/trapsession.py

```diff
@@ -38,8 +39,15 @@
 # Local constants
 IPADDR_SIZE = 4
 IP6ADDR_SIZE = 16
-IPADDR_OFFSET = 4
-IP6ADDR_OFFSET = 8
+IPADDR_OFFSET = 0
```

Comment:
Why 0? It is not affected by platform but has still changed?

Reply:
It changed because the code is now calculating the IP address offset relative to `SOCKADDR_DATA_OFFSET`, rather than from the start of an opaque struct. `SOCKADDR_DATA_OFFSET` will change dynamically based on the current platform (i.e. the actual offset varies both between IP address types and platforms, not just IP address type).